scripts/tile.py

```diff
@@ -115,12 +105,8 @@ def tiler(stack, date, mgrs):
                 with rasterio.open(name, "r+") as rst:
```

Comment:
At about L102 on the filename, @mattpaul had some suggestions at https://github.com/Clay-foundation/model/issues/35#issuecomment-1841585520 to:

1. Use underscore as the delimiter between MGRS/DATE/VERSION/COUNTER
2. Preface the VERSION with a v (e.g. `v0`)

Also mentioned at https://github.com/Clay-foundation/model/pull/44#discussion_r1399732116 that we could drop the hyphen in the date, i.e. YYYY-MM-DD becomes YYYYMMDD, but that didn't seem to have been applied in the initial batch.

Reply:
ok thanks will take that into consideration before next run.